README.md

```diff
@@ -176,11 +176,11 @@ jq.run('.', '/path/to/file.json', { output: 'string' }).then(console.log)
 |:----------------------------:|:---------------:|:-------:|
 | Read input stream into array | `true`, `false` | `false` |
 
-#### `slurp: 'true'`
+#### `slurp: true`
 
-Return the output as a **String**.
+Read input stream into array.
```

Comment:
I didn't read that before, sorry!

But stream here could be related to node Streams... and it's more like "piping" jsons?